lumicks/pylake/kymo.py

```diff
@@ -149,6 +149,11 @@ def _downsample_channel(self, n, xy, reduce=np.mean):
         # downsample exactly over the scanline time range
         min_times = self.timestamps[0, :].astype(np.int64)
         max_times = self.timestamps[-1, :].astype(np.int64)
+
+        # If the last line is incomplete, drop it!
+        if max_times[-1] == 0:
```

Comment:
Does that mean `0` cannot be a naturally occurring value?

Reply:
Yeah. Zero would be January 1st 1970, since they're nanoseconds since epoch :)